docs/src/components/landing/ToolpadFeaturesSwitcher.tsx

```diff
@@ -57,6 +61,12 @@ function ComponentItem({
             {description}
           </Typography>
         )}
+        {link && (
+          <Link href={link.toString()} underline="hover" variant="body2" target="_blank">
+            View docs
```

Comment:
```suggestion
            View the docs
```

to mimic X https://mui.com/x/